oci/oci.go

```diff
@@ -500,18 +513,20 @@ func (r *Runtime) UpdateStatus(c *Container) error {
 			if err != nil {
 				return fmt.Errorf("status code conversion failed: %v", err)
 			}
-			c.state.ExitCode = int32(statusCode)
+			c.State.ExitCode = int32(statusCode)
 		}
 	}
 
+	c.toDisk()
```

Comment:
One concern I have here is the amount of disk I/O that this will generate. I know we'll soon all have [RAM like storage ;)](http://www.intel.com/content/www/us/en/solid-state-drives/optane-solid-state-drives-dc-p4800x-series.html) but until then we may want to try saving a few IOPS...

So something I have not figured out from this implementation: Given that we store a `config.json` that also already contains all the k8s request metadata as an annotation, what are we missing that we'd get from `FromDisk()` ? Would storing the whole k8s request instead of only the metadata help ?